dapr/proto/runtime/v1/dapr.proto

```diff
@@ -672,4 +682,33 @@ message UnlockResponse {
   }
 
   Status status = 1;
-}
\ No newline at end of file
+}
+
+message WorkflowReference {
+  string InstanceID = 1;
+}
+
+message GetWorkflowRequest {
+  WorkflowReference WorkflowReference  = 1;
+  string WorkflowType = 2;
+  string WorkflowComponent = 3; 
+}
+
+message GetWorkflowResponse {
+  WorkflowReference WorkflowReference  = 1;
+  google.protobuf.Timestamp StartTime = 2;
+  map<string, string> Metadata = 3;
+}
+
+message StartWorkflowRequest {
+  WorkflowReference WorkflowReference = 1;
```

Comment:
Why using a message here? Do you think you'll need to add more properties in the future? Or can you just pass the workflow ID here?

Reply:
I do anticipate that we might add more properties in the future

Reply:
More properties under "WorkflowReference"? Or just in the "StartWorkflowRequest"?